constraint_utils/constraint_basics.py

```diff
@@ -883,25 +887,25 @@ def pLI(ht: hl.Table, obs: hl.expr.Int32Expression, exp: hl.expr.Float32Expressi
 def oe_confidence_interval(ht: hl.Table, obs: hl.expr.Int32Expression, exp: hl.expr.Float32Expression,
                            prefix: str = 'oe', alpha: float = 0.05, select_only_ci_metrics: bool = True) -> hl.Table:
     """
-    Indicate the bounds of confidence interval around the observed:expected ratio.
+    Determine the confidence interval around the observed:expected ratio.
     
-    For a given pair of observed and expected values, the function computes the density of the Poisson distribution
-    with fixed k (the observed number of variants) over a range of lambda values, which are given by the expected 
-    number of variants times a varying parameter ranging between 0 and 2. The cumulative density function of this 
-    function is computed and the value of the varying parameter is extracted at points corresponding to 5% and 95% 
-    to indicate the bounds of the confidence interval.
+    For a given pair of observed (`obs`) and expected (`exp`) values, the function computes the density of the Poisson distribution
+    (performed using Hail's `dpois` module) with fixed k (`x` in `dpois` is set to the observed number of variants) over a range of
+    lambda (`lamb` in `dpois`) values, which are given by the expected number of variants times a varying parameter ranging between
+    0 and 2. The cumulative density function of this function is computed and the value of the varying parameter is extracted at points
```

Comment:
"The cumulative density function of this function " doesn't sound quite right to me